src/main/java/com/google/sps/utility/AuthenticationUtility.java

```diff
@@ -37,9 +34,6 @@
   public static final String CLIENT_ID =
       "12440562259-mf97tunvqs179cu1bu7s6pg749gdpked.apps.googleusercontent.com";
```

Comment:
FFT: this would be really easy to hide using Zach's PR (https://github.com/googleinterns/step7-2020/pull/51)

Reply:
Tracking with issue #72